stl/inc/ranges

```diff
@@ -260,52 +266,55 @@ namespace ranges {
     template <bool _Enable, class _Rng, class _Derived>
     using _Cached_position_t = conditional_t<_Enable, _Cached_position<_Rng, _Derived>, view_interface<_Derived>>;
 
-    template <_Copy_constructible_object _Ty>
-    class _Copyable_box { // a simplified optional that augments copy_constructible types with full copyability
+    // a simplified optional that augments copy_constructible types with full copyability,
+    // and move_constructible types with full moveability.

```

Comment:
If the type is spelled `_Movable_box` then this should say:
```suggestion
    // and move_constructible types with full movability.
```

Reply:
augh, my brain has real difficulty with `movable` as opposed to `moveable` haha